ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -147,12 +168,55 @@ static StatsSnapshot convertQuotaToStatsSnapshot(Map<String, Map<String, Long>>
 
   /**
    * {@inheritDoc}
-   * Implementation in {@link BlobStoreStats} which returns the quota stats of a {@link BlobStore}. Size of delete
-   * records are not accounted as valid data size here.
+   * Implementation in {@link BlobStoreStats} which returns the all types of snapshots for a {@link BlobStore}. Size of
+   * delete records are not accounted as valid data size here. The formats of all snapshots are presented as follows.
+   * <pre>
+   *        AccountSnapshot             |             ContainerSnapshot
```

Comment:
minor: can you change these headers to reflect the `StatsReportType`? i.e. instead of "ContainerSnapshot" it should say "PARTITION_CLASS_REPORT".

Reply:
Added two comments. The detailed format of `PARTITION_CLASS_REPORT` and `ACCOUNT_REPORT` will be presented in next PR in HelixAggregator class. That's where the final two types of stats reports are generated for a single node.